plugins/action/ci_script.py

```diff
@@ -186,16 +187,17 @@ def run(self, tmp=None, task_vars=None):
             file_task.args.pop(arg_name, None)
 
         fnum = len(glob.glob(f"{output_dir}/ci_script_*"))
-        t_name = re.sub(r"([^\x00-\x7F]|\s)+", "_", self._task.name).lower()
+        t_name = re.sub(r"([^\x00-\x7F]|\s|-)+", "_", self._task.name).lower()
```

Comment:
since you're reworking that regexp, I think we should go with the "readable" `[^A-Za-z0-9_\]+` regexp - that would replace any non-alphanumeric (plus - and _) by "_", like I did in the run_hook.